notes/git-commit-history.md

```diff
@@ -0,0 +1,37 @@
+## COMMITS AND COMMITS HISTORY
+
+In Git, a commit is a fundamental concept and represents a snapshot of your project at a specific point in time. It is a way to save changes to your codebase, creating a record of what was modified, added, or deleted. Each commit has a **unique identifier** called a **hash**, and it contains information such as the author, timestamp, and a reference to the changes made.
+
+The history of commits in Git forms a branching, tree-like structure. When you create a new commit, it points to the previous commit(s), forming a chain. This series of commits creates a history that can be visualized as a graph.
+
+**Overview of commit history in Git:** 1.**Initial Commit:**
+The first commit in a Git repository is often referred to as the "initial commit." It marks the startingit init
+`git add .`
```

Comment:
for all bash scripts u have added use ```bash ```
https://youtrack.jetbrains.com/issue/IDEA-220660/Markdown-Shell-script-highlighting-for-bash-or-sh

Reply:
changed to:
```bash
git init
git add .
git commit -m "Initial commit"
```

I hope I understood you correctly.